description.md

```diff
@@ -20,6 +20,10 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Drills
 
+Drills are educational activities designed to reinforce learning through repetitive practice.
+The primary goal of drills is to promote mastery and automatization of skills or knowledge.
```

Comment:
```suggestion
The primary goal of drills is to promote mastery and automation of skills or knowledge.
```
Also, "automation" sounds rough and robotic. I suggest you use "internalisation" instead.